geomesa-tools/bin/install-hadoop-accumulo.sh

```diff
@@ -49,6 +52,7 @@ else
             "${base_url}commons-logging/commons-logging/${com_log_version}/commons-logging-${com_log_version}.jar"
             "${base_url}com/google/guava/guava/${guava_version}/guava-${guava_version}.jar"
             "${base_url}org/apache/commons/commons-vfs2/${commons_vfs2_version}/commons-vfs2-${commons_vfs2_version}.jar"
+            "${base_url}org/apache/htrace/htrace-core/${htrace_version}/htrace-core-${htrace_version}.jar"
```

Comment:
1.6 doesn't need it only 1.7 as far as I've noticed...also commons-vfs2 changes versions with 1.7